zookeeper/dataset.py

```diff
@@ -67,12 +70,16 @@ def validate_configuration(self):
 
     @property
     def name_version(self):
-        return f"{self.name}:{self.version}" if self.version else self.name
+        builder_config = (
+            f"/{self.builder_config}" if self.builder_config is not None else ""
+        )
+        version = f":{self.version}" if self.version is not None else ""
+        return self.name + builder_config + version
```

Comment:
One other thing we could do is to just pass the name, version and builder_config from the command line to the `tfds.load` that way you would do `--dataset voc/2012:4.0.0` from the command line. Though that might mess with the preprocessing. What do you think?

Reply:
Yes I am also fine doing it all in one string, you're right that there isn't really a reason to separate them

Reply:
Oh the only disadvantage might be that currently we do some validation to check that the name is valid. But I can either strip out everything after "/" and ":" or just not validate, because it will fail quickly.

Reply:
> But I can either strip out everything after "/" and ":" or just not validate, because it will fail quickly.

Yes, I'd be good with both approaches.